content/source/docs/enterprise/sentinel/mock.html.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: enterprise2
+page_title: "Mocking Terraform Sentinel Data - Sentinel - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-sentinel-mock"
+---
+
+# Mocking Terraform Sentinel Data
+
+Sentinel in Terraform Enterprise works with a very diverse range of data that is
+hard to predict the general structure of. Therefore, it's highly encouraged that
+you test your policies extensively before deploying them within your
+organization.
```

Comment:
This intro paragraph doesn't do much for me, and some of it is repeated below. I think you could omit it entirely and add "test your policies extensively before deploying them within your organization" to the next paragraph, and restructure it a little.

Reply:
Restructured, removed this paragraph, took the last sense as recommended, added it to the second paragraph, and then split that up.